### DISCUSSION TITLE
Unable to create user, already exists.

### DISCUSSION THREAD

**Discussion by adarshahd** [uxbox/uxbox#1745] [Q&A]
First of all, thank you very much for the much needed open source tool. 
I tried self hosting on an arm64 server and I am getting error creating user from the command line
`podman exec -it penpot_penpot-backend_1 ./manage.sh`
which gives the below error after providing user details.
`Unable to create user, already exists.`

User is not created. Any help is appreciated.

PenPot version: **1.12.2-beta**

docker-compose file (Adapted to my server OS RockyLinux with SELinux enabled)

version: "3.5"

services:
  penpot-postgres:
    image: postgres:alpine
    restart: always
    stop_signal: SIGINT
    environment:
      - POSTGRES_INITDB_ARGS=--data-checksums
      - POSTGRES_DB=penpot
      - POSTGRES_USER=penpot
      - POSTGRES_PASSWORD=penpot
    volumes:
      - ./data/postgres:/var/lib/postgresql/data:Z
  penpot-frontend:
    image: "penpotapp/frontend:latest"
    ports:
      - 9001:80
    volumes:
      - ./data/penpot_assets_data:/opt/data:Z
    env_file:
      - docker-compose.penpot.env
    depends_on:
      - penpot-backend
      - penpot-exporter
  penpot-backend:
    image: "penpotapp/backend:latest"
    volumes:
      - ./data/penpot_assets_data:/opt/data:Z
    depends_on:
      - penpot-postgres
      - penpot-redis
    env_file:
      - docker-compose.penpot.env
  penpot-exporter:
    image: "penpotapp/exporter:latest"
    environment:
      # Don't touch it; this uses internal docker network to
      # communicate with the frontend.
      - PENPOT_PUBLIC_URI=http://penpot-frontend
  penpot-redis:
    image: redis:alpine
    restart: always

**Comment by niwinz** [comment]
The user is created but is very probable that you are unable to login, because you need the `disable-secure-session-cookies` flag. This is happens because you are not using "localhost" nor https.
